### PR TITLE
Add in.bharatkalluri.shortcircuit

### PR DESCRIPTION
**Pull request by BharatKalluri**
**Description:** A developer scratchpad built for GNOME
**Project:** https://gitlab.gnome.org/Bharatkalluri/short-circuit

### CODE REVIEW

File: in.bharatkalluri.shortcircuit.json [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+                    "type" : "git",
+                    "url" : "https://gitlab.gnome.org/Bharatkalluri/short-circuit.git"
+                }
+            ]
+        }
+    ],
+    "build-options" : {
+        "env" : {
+        }
+    }
```
Comment (bilelmoussaoui) [inline in.bharatkalluri.shortcircuit.json:42]:
```suggestion
```

File: in.bharatkalluri.shortcircuit.json [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+        "/include",
+        "/lib/pkgconfig",
+        "/man",
+        "/share/doc",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "*.la",
+        "*.a"
+    ],
```
Comment (bilelmoussaoui) [inline in.bharatkalluri.shortcircuit.json:22]:
```suggestion
```
you don't need any of those in your case.

File: in.bharatkalluri.shortcircuit.json [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+            "name" : "shortcircuit",
+            "builddir" : true,
+            "buildsystem" : "meson",
+            "config-opts" : [
+                "--buildtype=release"
+            ],
+            "sources" : [
+                {
+                    "type" : "git",
+                    "url" : "https://gitlab.gnome.org/Bharatkalluri/short-circuit.git"
```
Comment (bilelmoussaoui) [inline in.bharatkalluri.shortcircuit.json:34]: This needs to be a tagged release or a specific commit

**Comment by bilelmoussaoui** [comment]
bot, build in.bharatkalluri.shortcircuit

**Comment by BharatKalluri** [comment]
> Build [23806 failed](https://flathub.org/builds/#/builders/32/builds/23806)

It looks like I missed release, screenshots and url tags in app data. Added all of them now (https://gitlab.gnome.org/Bharatkalluri/short-circuit/-/commit/5d64ce479120c71b8ee793130faf72fc4d85e473)

**Comment by bilelmoussaoui** [comment]
bot, build in.bharatkalluri.shortcircuit

**Comment by BharatKalluri** [comment]
Made some more changes to app data. Ran validate locally and it does not complain anymore. (It looks like even in the recent build, it complains about release and screenshot. Even though they do exist in the file :thinking: )

ref: https://gitlab.gnome.org/Bharatkalluri/short-circuit/-/blob/master/data/in.bharatkalluri.shortcircuit.appdata.xml.in

**Comment by hfiguiere** [comment]
> Made some more changes to app data. Ran validate locally and it does not complain anymore. (It looks like even in the recent build, it complains about release and screenshot. Even though they do exist in the file thinking )
> 
> ref: https://gitlab.gnome.org/Bharatkalluri/short-circuit/-/blob/master/data/in.bharatkalluri.shortcircuit.appdata.xml.in

But since you pull an older revision from git, they are not here.

This is the one it uses https://gitlab.gnome.org/Bharatkalluri/short-circuit/-/blob/be1fb0f122af90b9e83f56be35b69ff4bdf0d57a/data/in.bharatkalluri.shortcircuit.appdata.xml.in

**Comment by bilelmoussaoui** [comment]
bot, build in.bharatkalluri.shortcircuit

**Comment by BharatKalluri** [comment]
I am moving all UI to glade and doing a major refactor. I will reopen the issue once I am done with all that.

Thanks!

**Comment by BharatKalluri** [comment]
@bilelmoussaoui , Please have a look, I updated a bunch of things in the source repo and updated the commitID here.

**Comment by bilelmoussaoui** [comment]
bot, build in.bharatkalluri.shortcircuit

**Comment by nedrichards** [comment]
bot, build in.bharatkalluri.shortcircuit

**Comment by bilelmoussaoui** [comment]
bot, build in.bharatkalluri.shortcircuit

**Comment by bilelmoussaoui** [comment]
bot, build in.bharatkalluri.shortcircuit

**Pull request closed without merging**